content/essentials/understanding-jsx.mdx

```diff
@@ -0,0 +1,105 @@
+Consequently, parent-child relationships between elements are intuitive and easy to understand.
+
+### 2. Close all tags
+
+Self-closing tags are a must in JSX. 
+Unlike in HTML, where elements like `<input>`, `<img>`, or `<br>` don't require explicit closure, JSX requires consistent self-closing tags. 
+This ensures uniformity and helps avoid potential rendering issues.
+
+### 3.  Properties vs. attributes
+
+At first glance, HTML attributes and JSX props may seem simlar, but they serve different purposes and behave differently.
+Both offer ways to specify configurations or pass information, but they serve different purposes and behave differently in their respective realms: HTML for standard web content and JSX for Solid's component logic.
+
+#### HTML attributes
+
+[HTML attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes) are values set directly on HTML elements. 
+They supply additional information about an element, guiding its initial behavior and state. 
+These attributes are often translated into properties on DOM objects once the browser parses the HTML.
+
+In JSX files, you'll use HTML attributes much the same way you would in regular HTML, with a few key differences due to the blend of HTML and JavaScript:
+
+* Event listeners such as `onClick` are in camelCase.
+* You can dynamically specify a value, you can replace the `"` and `"` with `{` and `}`.
+
+```jsx
+<button class="myClass" onClick={handleClick}> Click me!</button>
+```
+
+<Aside>
+  **Note**: If you wish to pass objects in JSX, such as when you are doing inline styling, you will have to use double curly braces `{{ }}`.
+
+  ```jsx
+  <button style={{
+    color: 'red',
+    font-size: '2rem',
+  }}>
+    Click me!
+  </button>
+  ```
+</Aside>
+
+### JSX properties (props)
+
+JSX properties, commonly known as "props", enable the passing of data and configurations to components in your application.
+They are the bridge between the component and the data it requires, providing seamless data flow and dynamic interaction.
+
+#### Core concepts
+
+* **Static vs. dynamic attributes**: 
+In Solid's JSX, static attributes are integrated directly into the HTML template that is cloned. 
+Dynamic attributes, however, are not hard-coded into the template, instead relying on state and other reactive data sources. 
+This allows for content or styling of an element to change in response to interactions or other changes in the application's data.
+
+* **Dynamic data transfer**: 
+Props play a pivotal role when using dynamic data within your application. 
+They empower components to react in real-time to data changes, resulting in a responsive application.
+
+<Aside>
+  Expressions, whether fixed or dynamic, get applied in the order defined within the JSX. 
+  This sequence works for a wide range of DOM elements.
+  However, be cautious with elements like input types with `type='range'` - the order matters and can influence the element's behavior.
```

Comment:
```suggestion
  However, be cautious with elements like input types with `type='range'` - the order matters and it can influence the element's behavior.
```